mcs_benchmark_data/infile_labels_benchmark_file_names.py

```diff
@@ -0,0 +1,13 @@
+from typing import NamedTuple
+from dataclasses import dataclass
+
+from mcs_benchmark_data._benchmark_file_names import _BenchmarkFileNames
+
+
+@dataclass(frozen=True)
+class InfileLabelsBenchmarkFileNames(_BenchmarkFileNames):
+    dev_labels: str

```

Comment:
Docstring indicating what this class is for, since "infile" is not a common term.